### ISSUE TITLE
Misuse of aria-live causes unexpected announcements

### ISSUE THREAD

**Issue by Woebin** [JohnXLivingston/peertube-plugin-livechat#721]
**Describe the bug**
This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 4.1.3 (AA) Status Messages](https://www.w3.org/WAI/WCAG22/Understanding/status-messages.html)

The chat widget does not work as expected with screen readers. For example, when using VoiceOver on MacOS and the Safari browser:
- The first messages in the chat history get announced when submitting a new message, due to the entire chat history having the `aria-live="polite"` attribute.
- The "menu" button next to each message also gets announced, seemingly multiple times per message.

The following video demonstrates this behavior:
https://github.com/user-attachments/assets/67db10f2-1e53-4064-8160-5addc6d684de

**To Reproduce**
1. Launch a screen reader, VoiceOver on MacOS is where this issue has been observed.
2. Send a message in chat.
3. Hear VoiceOver announce more than it should.

**Expected behavior**
VoiceOver should announce new messages in the chat, not the entire chat history.

**Suggestion**
Look into [aria-live](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-live), [aria-atomic](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-atomic) and [aria-relevant](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-relevant), how to implement it correctly, and figure out which options are best for you. You most likely want to only announce the most recent message, as it appears in the chat, as well as maybe edits to existing messages.

Test the interface with screen readers to ensure that it behaves correctly.

**Server:**
 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: 12.0.3

**Desktop:**
 - OS: MacOS 15.5
 - Browser: Google Chrome Version 137.0.7151.120 (Official Build) (arm64)

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).
